src/components/panels/NightwavePanel.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <HubPanelWrap :title="headertext">
+    <b-list-group>
+      <b-list-group-item :style="styleObject" v-for="(challenge, index) in nightwave.activeChallenges" :key="challenge.id"
+      v-bind:class="{ 'list-group-item-borderless': index !== nightwave.activeChallenges.length - 1,
+        'list-group-item-borderbottom': index === nightwave.activeChallenges.length - 1 }">
+        <span class="pull-left">
+          <HubImg
+            :src="challenge.isDaily ? daily : (challenge.isElite ? elite : weekly)"
```

Comment:
This looks like an opportunity to make different subclasses of HubImg for each type of challenge.

Reply:
it's different sources, which the hubimg is meant to handle all of. doesn't really seem necessary to wrap it, because we'd still have to perform the same logic in the end, just with components instead of sources.